awx/ui_next/src/screens/Credential/CredentialList/CredentialList.jsx

```diff
@@ -153,17 +153,17 @@ function CredentialList({ i18n }) {
                   itemsToDelete={selected}
                   pluralizedItemName={i18n._(t`Credentials`)}
                 />,
-                canAdd && (
-                  <ToolbarAddButton key="add" linkTo="/credentials/add" />
-                ),
+                ...(canAdd
```

Comment:
Is this that the old way would make it [<ToolbarDeleteButton>, undefined] and the new way doesn't?

Reply:
Yes, the old way would pass [ <AddButton />, null ]. The spread operator will "filter" out the empty array.